core/src/tvu.rs

```diff
@@ -183,7 +178,7 @@ impl Tvu {
             exit: exit.clone(),
             subscriptions: subscriptions.clone(),
             leader_schedule_cache: leader_schedule_cache.clone(),
-            slot_full_senders: vec![blockstream_slot_sender],
+            slot_full_senders: vec![],
```

Comment:
Is `slot_full_senders` used for anything else? I didn't find anything.
This can be removed as well?

Reply:
No other users, I almost deleted it but thought maybe we might want it to map blockstream into RPC one day.  Donno, wdyt?  Remove it all?

Reply:
My vote is to remove. We know how to find this again if we want to reimplement it later.